web/modules/custom/rsvplist/src/EnablerService.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\rsvplist;
+
+use Drupal;
+use Drupal\Core\database\Connection;
+use Drupal\node\Entity\Node;
+use Exception;
+
+/**
+ * EnablerService Class.
+ */
+class EnablerService {
+
+  /**
+   * @var Connection $database_connection
+   *   A database connection object.
+   */
+  protected Connection $database_connection;
```

Comment:
Class property $database_connection should use lowerCamel naming without underscores

Reply:
i have some doubt regarding it  as at many places we are using camel case and some places we are using the snake case

Reply:
Local variables should be snake_case and Class variables should be camelCase